packages/docs/stories/styles/table-cell.stories.ts

```diff
@@ -0,0 +1,186 @@
+
+const meta: Meta = {
+  args: overrideArgs([
+    {
+      name: 'default',
+      type: 'slot',
+      value: 'Cell content',
+    },
+  ], defaultArgs),
+  argTypes,
+  component: 'syn-table-cell',
+  parameters: {
+    design: generateFigmaPluginObject('19479-134433'),
+    docs: {
+      description: {
+        component: generateStoryDescription('table', 'default'),
+      },
+    },
+  },
+  title: 'Styles/syn-table-cell',
+};
+export default meta;
+
+export const Default: StoryObj = {
+  parameters: {
+    controls: {
+      disable: false,
+    },
+    docs: {
+      description: {
+        story: generateStoryDescription('table', 'default'),
+      },
+    },
+  },
+  render: (args: unknown) => html`
+  <table class="syn-table">
+    <tbody>
+      <tr>
+        ${renderStyles(({ ...(args as RenderArgs), sticky: true } as RenderArgs), 'td')}
+      </tr>
+    </tbody>
+  </table>
+  <style>
+    .sticky {
+      position: sticky;
+    }
+  </style>
+  `,
+};
+
+export const Header: StoryObj = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('table', 'header'),
+      },
+    },
+  },
+  render: () => html`
+    <table class="syn-table">
+      <thead>
+        <tr>
+          <th class="syn-table-cell--header">Cell header</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td class="syn-table-cell">Cell content</td>
+        </tr>
+      </tbody>
+    </table>
+  `,
+};
+
+export const Alternating: StoryObj = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('table', 'alternating'),
+      },
+    },
+  },
+  render: () => html`
+    <table class="syn-table">
+      <tbody>
+        <tr>
+          <td class="syn-table-cell">Cell content</td>
+        </tr>
+        <tr>
+          <td class="syn-table-cell syn-table-cell--bg-neutral-50 ">Cell content</td>
+        </tr>
+        <tr>
+          <td class="syn-table-cell">Cell content</td>
+        </tr>
+        <tr>
+          <td class="syn-table-cell syn-table-cell--bg-neutral-50 ">Cell content</td>
+        </tr>
+      </tbody>
+    </table>
+  `,
+};
+
+export const Border: StoryObj = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('table', 'border'),
+      },
+    },
+  },
+  render: () => html`
+    <table class="syn-table">
+      <tbody>
+        <tr>
+          <td class="syn-table-cell syn-table-cell--border-top">Border Top</td>
+        </tr>
+        <tr>
+          <td class="syn-table-cell syn-table-cell--border-start">Border Start</td>
+        </tr>
+        <tr>
+          <td class="syn-table-cell syn-table-cell--border-end">Border End</td>
+        </tr>
+        <tr>
+          <td class="syn-table-cell syn-table-cell--border-bottom">Border Left</td>
+        </tr>
+      </tbody>
+    </table>
+  `,
+};
+
+export const ScrollingBehavior: StoryObj = {
```

Comment:
Does this include any logic? I saw that a `sticky` class is provided, however it is not used here.

Reply:
The sticky class is used in line 159 and 160.
If the shadow should be visible, we need either a `position: relative` or a `position: sticky` on the table-cell with the shadow, otherwise it is not visible / on the correct place

Reply:
As someone that has a lot of bad experience with tables and sticky columns: Is `position: sticky` working on table cells with all browsers now? Did not know that.

Reply:
I tested it with all current browsers and on can I use I found this: 
<img width="873" alt="Bildschirmfoto 2024-07-02 um 11 21 39" src="https://github.com/synergy-design-system/synergy-design-system/assets/115139104/e237ede6-579a-4ebb-a512-dcc304351012">

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/2d8ec146-e45e-41f6-b578-d20e9d13245b)


It seems to work for all, for a while now

Reply:
Thats great news (that are not so new it seems... I better polish up my table css skills ;))